launcher/game/translations.rpy

```diff
@@ -27,6 +27,8 @@ init python:
     if persistent.translate_language is None:
         persistent.translate_language = "english"
 
+    persistent.extract_language = persistent.extract_language or "None"
```

Comment:
I think this would better be placed in the "extract_dialogue" label, or even in the "extract_dialogue" screen, so that it's updated every time the variable is read, used or set.